Mjolnir/src/Subsystems/Intake.cpp

```diff
@@ -8,35 +8,44 @@
 // update. Deleting the comments indicating the section will prevent
 // it from being updated in the future.
 
-
-#include "CANTalon.h"
-
-
 #include "Intake.h"
 #include "../RobotMap.h"
+#include "Commands/SetIntake.h"
+
+
 // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=INCLUDES
 // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=INCLUDES
 
 // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
 // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
 
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
-
+const char Intake::kSubsystemName[] = "Intake";
+std::shared_ptr<Intake> Intake::self;
+std::shared_ptr<Intake> Intake::getInstance() {
+	if (! self) {
+		self = std::shared_ptr<Intake>(new Intake);
+	}
+	return self;
+}
 Intake::Intake() : Subsystem("Intake") {
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-    leftIntake = RobotMap::intakeleftIntake;
-    rightIntake = RobotMap::intakerightIntake;
+	static std::shared_ptr<WPI_TalonSRX> leftIntake;
```

Comment:
I think we should not get into this indent habit. Besides, Robot builder is not going to be editing this code any longer. So the comments don't apply.